openedx/features/survey_report/models.py

```diff
@@ -0,0 +1,31 @@
+"""
+Survey Report models.
+"""
+
+from django.db import models
+from jsonfield import JSONField
+
+
+class SurveyReport(models.Model):
+    """
+    This model stores information to automate the way of gathering impact data from the openedx project.
+
+    .. no_pii:
+    """
+    courses_offered = models.IntegerField()
+    learners = models.IntegerField()
+    registered_learners = models.IntegerField()
+    enrollments = models.IntegerField()
+    generated_certificates = models.IntegerField()
+    extra_data = JSONField(
+        blank=True,
+        default={},
+        help_text="Extra information for instance data",
+    )
+    request_details = models.CharField(max_length=255, null=True, blank=True)
+    created_at = models.DateTimeField(auto_now=True)
```

Comment:
It might be useful to also track how long the report took to generate, even broken down by query if possible. It's possible that the overhead of this survey tool could grow in ways that we don't expect, discouraging people from running it on larger sites.

Reply:
@ormsbee we have already removed this, I'm gonna add this to the future upload model.